gluoncv/auto/estimators/image_classification/image_classification.py

```diff
@@ -417,15 +437,29 @@ def _evaluate(self, val_data):
                 val_data.transform_first(transform_test),
                 batch_size=self._cfg.valid.batch_size, shuffle=False, last_batch='keep',
                 num_workers=self._cfg.valid.num_workers)
-        for _, batch in enumerate(val_data):
-            data, label = self.batch_fn(batch, self.ctx)
-            outputs = [self.net(X.astype(self._cfg.train.dtype, copy=False)) for X in data]
-            acc_top1.update(label, outputs)
-            acc_top5.update(label, outputs)
+            
+        if metric_name == 'rmse':
+            rmse_metric = mx.metric.RMSE()
+            for _, batch in enumerate(val_data):
+                data, label = self.batch_fn(batch, self.ctx)
+                outputs = [self.net(X.astype(self._cfg.train.dtype, copy=False)) for X in data]
+                rmse_metric.update(label, outputs)
+
+            _, val_score = rmse_metric.get()
+            return -val_score
```

Comment:
I do not think we should return the negative score. This can be confusing. (I know that AG will return the negative score but that should be fixed).

Reply:
This is an easy way to replace best_acc in early stopper.